src/handler/batchPlateCreation.ts

```diff
@@ -78,26 +72,19 @@ export const handler = async (batchPlateRecords: BatchPlateData[]) => {
           documentName: DocumentName.MINISTRY,
           recipientEmailAddress: '',
         };
-
         logger.debug(JSON.stringify(plateSqsPayload));
 
         await addToSqs(plateSqsPayload, process.env.DOC_GEN_SQS_QUEUE ?? '');
 
         numberOfSqsAdded += 1;
-      } catch (err) {
-        logger.error(`${systemNumber}, ${createdTimestamp}, ${JSON.stringify(err)}`);
+      } catch (err: unknown) {
```

Comment:
why are we using unknown and casting the entire object to a string here? can we not catch it as an `Error` and use `err.message`?

Reply:
During testing, when i left it as it was - it seemed to be producing the logs as error: {}. However, with the casting it seemed to resolve that issue

Reply:
```
catch (error: unknown) {
			if (error instanceof CustomError) {
				this.logger.error(error.message);
			} else if (error instanceof Error) {
				this.logger.error(ErrorEnum.ERROR_OCCURRED, error.message);
			} else {
				this.logger.error(ErrorEnum.UNKNOWN, JSON.stringify(error));
			}
		}
```
this is how it was handled recently in central docs, which does stringify the entire object but only after some type handling. This could be left for logging ticket though